lib/spack/spack/environment/environment.py

```diff
@@ -474,14 +474,14 @@ def __contains__(self, spec):
 
         return True
 
-    def specs_for_view(self, concretized_specs):
+    def specs_for_view(self, root_specs):
```

Comment:
Just a comment. Can we use:
```suggestion
    def specs_for_view(self, concretized_root_specs):
```
here and below? That makes clear that we expect the concretized version of the root specs in these calls (as opposed to the ones that appear in `spack.yaml`)